src/Core/System.php

```diff
@@ -212,7 +212,10 @@ public static function createGUID($size = 16, $prefix = '')
 	 */
 	public static function processID($prefix)
 	{
-		return uniqid($prefix . ':' . str_pad(getmypid() . ':', 8, '0') . ':');
+		// We aren't calling any other function here.
```

Comment:
What other function?

Reply:
I think this is a hint for me to not use the GUID function here ;-).
@annando I won't use it, because as you already found out, there'll be an infinit loop if something goes wrong here :-)

Reply:
I applied the changes I proposed in the PR from @nupplaphil - and created an endless loop, since that function called another function that does some logging - which resulted in calling this function that called another function ...